Simulator/Edges/AllEdges.h

```diff
@@ -32,7 +32,7 @@
 #include "IAllEdges.h"
 
 // cereal
-#include <ThirdParty/cereal/types/vector.hpp>
+#include <../ThirdParty/cereal/types/vector.hpp>
```

Comment:
Why using angle brackets, instead of quotes?

Reply:
Yeah, you're right. The brackets should only be used when locating library files outside of the project. I will change that.